dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcTeiAnalyticsTableManager.java

```diff
@@ -120,7 +118,6 @@ public JdbcTeiAnalyticsTableManager( IdentifiableObjectManager idObjectManager,
         new AnalyticsTableColumn( quote( "trackedentityinstanceid" ), INTEGER, NOT_NULL,
             "tei.trackedentityinstanceid" ),
         new AnalyticsTableColumn( quote( "trackedentityinstanceuid" ), CHARACTER_11, NOT_NULL, "tei.uid" ),
-        new AnalyticsTableColumn( quote( "trackedentitytypeuid" ), CHARACTER_11, NOT_NULL, "tet.uid" ),
```

Comment:
We don't need a column with trackedEntityType, since each row will hold the same value